libs/labelbox/CHANGELOG.md

```diff
@@ -1,4 +1,28 @@
 # Changelog
+# Version 3.75.0 (2024-07-10)
+
+## What's Changed
+* [PLT-0] Predictions missing by @adrian-chang in https://github.com/Labelbox/labelbox-python/pull/1695
+* Use correct project type in tests/integration/test_project_model_config.py by @vbrodsky in https://github.com/Labelbox/labelbox-python/pull/1697
+* [PLT-1151] Vb/model setup complete by @vbrodsky in https://github.com/Labelbox/labelbox-python/pull/1685
```

Comment:
Looks like these are generated from PR comments... should we have it a little more polished? I usually do

Reply:
They are, and I'm trying to minimize the time spent on changelog notes. Some of it is useful (the URLS) although some of it is not so useful (random / odd PR titles).